### PR TITLE
Matthijs/clouddocs/awstutorial

### PR DESCRIPTION
**Pull request by matthijskrul**
## Summary of the changes / Why this is an improvement
- Updated AWS tutorial for changed user flow
- Screenshots added/updated
- Reflect introduction of CrateDB Cloud Contract
- Misc

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Using the CrateDB Cloud offer on AWS Marketplace
 
 To subscribe to CrateDB Cloud's offering on AWS Marketplace, find the CrateDB
 Cloud offer page. You can either search "CrateDB Cloud" in the search bar at
-the top of the AWS Marketplace front page, or follow `this link`_ to the offer.
+the top of the AWS Marketplace front page, or follow `this link to the offer`_.
```
Comment (nomicode) [inline docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst]: I noticed you improving some of the links in this doc! looks good!

one minor nit: I said it's a better style if links use noun phrases, and using "this link" (or similar) DOES technically count (haha) but then the noun is pointing to the link itself and not the destination the link takes you to. perhaps think of the link text like the label you would put on a map? it should describe the location, not the method of travel ^_^

so in this case, how about:

> You can either search "CrateDB Cloud" in the search bar at the top of the AWS Marketplace front page or go directly to [the offer](http://example.com/).

**Review by nomicode — APPROVED** [review]
lgtm

left one small comment that you might wanna address. (will also add the feedback below to the new link style guide)

File: docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst [hunk 2]
```diff
@@ -49,17 +49,36 @@ configuration on the CrateDB Cloud end.
...
+.. _signup-aws-contract:
+
+Using the CrateDB Cloud Contract on AWS Marketplace
+===================================================
+
+Alternatively, you can avail yourself of the possibility of an annual contract
+for our cloud service through the CrateDB Cloud Contract. On AWS Marketplace,
+the contract is a separate product for subscription purposes. You can access it
+by going to the `CrateDB Cloud Contract landing page`_.
+
```
Comment (lukas-bernert) [inline docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst:61]: Can we add a note, that if customer's plan to purchase higher volumes they should get in contact with sales@crate.io first to negotiate a private offer?

File: docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst (hunk 1 above)
Comment (nomicode) [reply docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst]: updated https://github.com/crate/crate-docs/blob/master/style/hypertext.rst

File: docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst (hunk 2 above)
Comment (matthijskrul) [reply docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst:61]: That is also mentioned on the Subscription Plans page, but I can certainly add it here too

**Pull request merged**